packages/odyssey-react-mui/src/theme/temp.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+export type Temp = {
+  borderWidth: string;
```

Comment:
Two questions:

1. Does this result in `theme.temp.borderWidth` as an available variable?
2. Does this support children? for ex: `theme.border.style.ui`

Reply:
1. Yes.
2. Yes. Just make another object and assign typed properties to it.